### ISSUE TITLE
Actors sheets sometimes show HP as 0

### ISSUE THREAD

**Issue by yochaigal** [yochaigal/Cairn-FoundryVTT#6]
HP is sometimes 0, and cannot be changed. It eventually goes away, though I've seen it come back as well.

See screenshot below:

![HP bug](https://user-images.githubusercontent.com/817479/100458378-a9acd700-3091-11eb-9c48-7ec21a82b211.png)

**Comment by smcabrera** [comment]
I finally figured out what's going on with this. This happens if you have more than two bulky items and is a holdover from Electric Bastionland, in which by carrying more than 2 bulky items (or more than 1? I forget) you are reduced to 0 hit points. 

I went back through and looked at the latest Cairn rules and noted that this is not how bulky items work in Cairn. It appears that in Cairn "bulky" is basically just a tag meaning that the item takes up two slots instead of 1. Meanwhile, the rule about reducing your HP to 0 happens if you are carrying a full inventory. 

Here's what I can do:
1. **Remove bulky check**: I can remove the bulky check and that should fix this issue in the case that it's happening for people with more than two bulky items, which is just a vestigial artifact from Electric Bastionland. 
2. **Signal encumbrance better**: This is still what's going to happen if someone is "encumbered", which is to say their used item slots are greater than or equal to 10. For that--we might want to think about whether there's a way that we can signal that the person is encumbered and that's what's affecting their health, and include that somewhere on the character sheet since right now there is no direct link shown and it can be confusing / look like a bug. 

Separately, I wonder I wonder if it's worth thinking about removing the bulky keyword altogether. My thinking is that there are already some items, which you've changed to be between 0 and 1 slots. In Electric Bastionland "bulky" is a tag that is used in place of item slots, but if the game has item slots it seems like it might be simpler for those bulky items to simply be items with "slots" equal to 2. Especially since you're going to have to label the other items as "slots 1/2" or whatever. Just a thought. 

I can do #1 right away, #2 may take a bit more time to make some styling that looks reasonable.
